staking/overview.md

```diff
@@ -15,6 +15,21 @@ The collators' set (nodes that produce blocks) are selected based on their stake
 
 Collators (and token holders if they nominate) have a stake in the network. The top N collators by staked amount are chosen to produce blocks with a valid set of transactions, where N is a configurable parameter. Part of each block reward goes to the collator that produced the block, who then shares it with the nominators considering their percental contributions towards the collator's staked. In such a way, network members are incentivized to stake tokens to improve the overall security.
 
+## Quick Reference {: #quick-reference }
+
+* Minimum Nomination Amount: {{ networks.moonbase.staking.min_nom_stake }} MOVR/DEV
+* Max Eligible Nominators Per Collator: Top {{ networks.moonriver.staking.max_nom_per_col }} nominators by size of stake per collator are eligible for staking rewards (nominators not in the top {{ networks.moonriver.staking.max_nom_per_col }} do not receive any rewards)
+* Bonding Duration: {{ networks.moonriver.staking.bond_lock }} rounds ({{ networks.moonriver.staking.round_blocks }} blocks/round, time per round is approximately 1 hour)
+* Unbonding Duration: {{ networks.moonriver.staking.bond_lock }} rounds 
+* Reward Payout Time: Rewards are distributed automatically to the free balance {{ networks.moonriver.collator_timings.rewards_payouts.rounds }} rounds after they are earned
+* Aggregate Nominator Rewards: 50% of total inflation
+* Nominator Rewards: Variable; it's the aggregate nominator rewards distributed over all eligible nominators, taking into account the relative size of stakes; [More Information](/staking/overview/#reward-distribution)
+* Collator Commission: Fixed at 20% of total inflation, separate from the nominator reward pool
+* Slashing: No slashing currently (can be changed with governance); collator rewards are paid as part of produced blocks, so offline or poorly performing collators that don’t produce blocks will pay out less or no rewards
+* Collator Information: [Link1](https://moonriver.subscan.io/validator) [Link2](https://moonbeam-explorer.netlify.app/stats/miners/)
+* Managing Staking and Nomination Related Actions: [Link](https://apps.moonbeam.network/moonriver)
```

Comment:
The formatting for this list needs to be updated. For an example of a formatted list, check out `snippets/text/staking/staking-definitions`

But in general the list should be formatted something like this:
- **Item one** - this is something
- **Item two** - this is something else

Reply:
Thanks. I've removed the snippets now and changed the list formatting. 

I've also slightly modified the parts which used to have the Moonriver launch snippet to make it more explicit the relevant staking information is applicable to both Moonriver and Moonbase Alpha to account for removing the Moonriver launch snippet.